python/lsst/ts/externalscripts/auxtel/latiss_base_align.py

```diff
@@ -0,0 +1,788 @@
+                description: >-
+                    Optional configuration section. Find a target to perform CWFS in the given
+                    position and magnitude range. If not specified, the step is ignored.
+                properties:
+                  az:
+                    type: number
+                    description: Azimuth (in degrees) to find a target.
+                  el:
+                    type: number
+                    description: Elevation (in degrees) to find a target.
+                  mag_limit:
+                    type: number
+                    description: Minimum (brightest) V-magnitude limit.
+                  mag_range:
+                    type: number
+                    description: >-
+                        Magnitude range. The maximum/faintest limit is defined as
+                        mag_limit+mag_range.
+                  radius:
+                    type: number
+                    description: Radius of the cone search (in degrees).
+              track_target:
+                type: object
+                additionalProperties: false
+                required:
+                  - target_name
+                description: >-
+                    Optional configuration section. Track a specified target to
+                    perform CWFS. If not specified, the step is ignored.
+                properties:
+                  target_name:
+                    description: Target name
+                    type: string
+                  icrs:
+                    type: object
+                    additionalProperties: false
+                    description: Optional ICRS coordinates.
+                    required:
+                      - ra
+                      - dec
+                    properties:
+                      ra:
+                        description: ICRS right ascension (hour).
+                        type: number
+                        minimum: 0
+                        maximum: 24
+                      dec:
+                        description: ICRS declination (deg).
+                        type: number
+                        minimum: -90
+                        maximum: 90
+              filter:
+                description: Which filter to use when taking intra/extra focal images.
+                type: string
+                default: empty_1
+              grating:
+                description: Which grating to use when taking intra/extra focal images.
+                type: string
+                default: empty_1
+              exposure_time:
+                description: The exposure time to use when taking intra/extra focal images (sec).
+                type: number
+                default: 30.
+              acq_exposure_time:
+                description: The exposure time to use when taking an in focus acquisition image(sec).
+                type: number
+                default: 5.
+              take_detection_image:
```

Comment:
So this was used for the WEP testing parts before, correct? Is it still required? 
If so, maybe add a sentence in the description saying why it exists and when to use it.

Reply:
Yes, I added this for early WEP testing. In my implementation I ended up using the same "ISR" and detection code we use for the old cwfs code so, it is not used for processing the data. I though about leaving it just because it might be a nice debugging tool eventually. But it is right that it adds unnecessary complexity, so maybe I should remove it. 

I'm not sure, what do you think? remove or keep it for debugging purposes and add a note about it?

Reply:
I think you can leave it, just add a sentence to the description. 
Something like, "Used for early WEP testing but maintained for possible debugging scenarios"